docs/tutorial/using-grpc-plugins.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: "Building and deploying gRPC plugins"
+description: "Learn how to use gRPC plugins with Cosmo & Cosmo Router"
+icon: plug
+---
+
+## Prerequisites
+
+- [Cosmo CLI](/cli/intro) installed and configured
+
+## Overview
+
+This tutorial shows you how to create a gRPC plugin subgraph from scratch using the Cosmo CLI.
```

Comment:
Either it is possible for the user to copy / paste the files or the full example needs to be publicly available.

Reply:
Added the full implementation to an appendix accordion